FapgansControleBot/Models/gans.py

```diff
@@ -1,8 +1,19 @@
-from sqlalchemy import Column, Integer, String, DateTime
+import datetime
 
+from sqlalchemy import Column, Integer, String, DateTime, ForeignKey
+from sqlalchemy.orm import relationship
 
-class Gans:
+from FapgansControleBot.Repository.database import Base
+
+
+class Gans(Base):
     __tablename__ = 'gans'
     gans_id = Column(Integer, primary_key=True)
-    user_id = Column(Integer)
+    user_id = Column(ForeignKey('users.user_id'))
+    credit_id = Column(ForeignKey('credits.credit_id'))
```

Comment:
I don't fully understand the purpose of this Gans entity/table now it has a relation to both user and credit...
Overall it looks bit bit too complex to me as well, but maybe I'm missing something. 

Can't we just record facts (user X has sent fapgans on timestamp Y). Then we have application logic which determines if the user should be reported or not. 
Then we only need 1 table of all the facts, and 1 table for current price. Or am I thinking too simple now?